versioned_docs/version-0.4.0/kubeslice-cli/kubeslice-cli-demo.mdx

```diff
@@ -78,9 +79,22 @@ ks-w-2   54m
 You have successfully installed the KubeSlice Controller on the controller cluster and Slice Operator on the worker cluster.
 :::
 
-## Creating a Slice
 
-After successfully installing the KubeSlice Controller and KubeSlice Worker, you can create a slice to onboard your application.
+After successfully installing KubeSlice Controller and the Slice Operator on the worker clusters. The `kubeslice-cli install --profile=minimal-demo` 
+command creates a slice called `demo`. To validate the demo slice, see [Validating the Slice](#validating-the-slice).
+
+On the `demo` slice, you can now onboard the iperf application. To onboard the application on the slice, see
+[Deploying the iPerf Application](#deploying-the-iperf-application). 
+
+You can also use the kubeslice-cli command to create a new slice where you can onboard application. To create a slice on your demo setup, 
+follow these steps.
+
+
+## Creating a Slice
```

Comment:
If this is the demo topic, then the demo (minimal | full) both would create a slice right? 

@uma-kt / @priyank-upadhyay  - do we need to describe create a slice topic here or can we skip describing creating a slice and move to onboarding namespaces?

Reply:
@veenamj-avesha  yes minimal and full demo creates a slice. Creating a slice is required if the user wants to try creating a new slice.   As per Eric's comment in the group, we can keep that step if the user wants to create a slice.

Reply:
I will add a paragraph or two lines explaining the difference between full and minimal demo.

Reply:
added the below note:
:::info
Skips this step if you do not want to create a new slice and continue onboarding the application on the `demo` slice.
:::